### DISCUSSION TITLE
Is there any way to apply a shader onto a camera?

### DISCUSSION THREAD

**Discussion by SSoup64** [HaxeFlixel/flixel#2610] [General]
I have made an OpenFL shader and I want to apply to a camera, I have no found to way to do that thus far.
All help is appreciated.

**Comment by Geokureli** [comment]
https://api.haxeflixel.com/flixel/FlxCamera.html#setFilters
like so:
```hx
import openfl.filters.ShaderFilter;
//...

FlxG.camera.setFilters([new ShaderFilter(new MyCustomShader())]);
```
